bigtable/client/mutations.h

```diff
@@ -74,17 +75,54 @@ constexpr std::int64_t ServerSetTimestamp() { return -1; }
  */
 
 /// Delete only within the timestamp range provided.
+template <typename Rep1, typename Period1, typename Rep2, typename Period2>
```

Comment:
Please include Doxygen documentation for `Rep1`, `Period1` and company.  You can use the examples in the `bigtable/client/filters.h` file.